system/lib/libcxxabi/src/cxa_default_handlers.cpp

```diff
@@ -13,11 +13,11 @@
 #include "abort_message.h"
 #include "cxxabi.h"
 #include "cxa_handlers.h"
-#include "cxa_exception.h"
 #include "private_typeinfo.h"
 #include "include/atomic_support.h"
 
 #if !defined(LIBCXXABI_SILENT_TERMINATE)
+#include "cxa_exception.h"
```

Comment:
is this a divergence from upstream? (if so then an `XXX EMSCRIPTEN` comment?)

Reply:
Done